sample/AspNetFullFrameworkSampleApp/Web.config

```diff
@@ -1,317 +1,305 @@
 	</configSections>
 	<connectionStrings>
-		<add name="db" connectionString="data source=:memory:" providerName="System.Data.SQLite" />
+		<add name="db" connectionString="data source=:memory:" providerName="System.Data.SQLite"/>
 	</connectionStrings>
-  <appSettings>
-    <add key="webpages:Version" value="3.0.0.0" />
-    <add key="webpages:Enabled" value="false" />
-    <add key="ClientValidationEnabled" value="true" />
-    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
-    <add key="ElasticApm:MaxQueueEventCount" value="9128" />
-  </appSettings>
-  <system.web>
-    <compilation debug="true" targetFramework="4.6.1" />
-    <httpRuntime targetFramework="4.6.1" />
-    <httpModules>
-      <add name="TelemetryCorrelationHttpModule" type="Microsoft.AspNet.TelemetryCorrelation.TelemetryCorrelationHttpModule, Microsoft.AspNet.TelemetryCorrelation" />
-      <add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" />
-    </httpModules>
-  </system.web>
-  <runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+	<appSettings>
+		<add key="webpages:Version" value="3.0.0.0"/>
+		<add key="webpages:Enabled" value="false"/>
+		<add key="ClientValidationEnabled" value="true"/>
+		<add key="UnobtrusiveJavaScriptEnabled" value="true"/>
+		<add key="ElasticApm:MaxQueueEventCount" value="9128"/>
+	</appSettings>
+	<system.web>
+		<compilation debug="true" targetFramework="4.6.1"/>
+		<httpRuntime targetFramework="4.6.1"/>
+		<httpModules>
+			<add name="TelemetryCorrelationHttpModule" type="Microsoft.AspNet.TelemetryCorrelation.TelemetryCorrelationHttpModule, Microsoft.AspNet.TelemetryCorrelation"/>
+			<add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web"/>
+		</httpModules>
+	</system.web>
+	<system.webServer>
+		<modules>
+			<remove name="ElasticApmModule"/>
+			<add name="ElasticApmModule" type="Elastic.Apm.AspNetFullFramework.ElasticApmModule"/>
+			<remove name="TelemetryCorrelationHttpModule"/>
+			<add name="TelemetryCorrelationHttpModule" type="Microsoft.AspNet.TelemetryCorrelation.TelemetryCorrelationHttpModule, Microsoft.AspNet.TelemetryCorrelation" preCondition="managedHandler"/>
+			<remove name="ApplicationInsightsWebTracking"/>
+			<add name="ApplicationInsightsWebTracking" type="Microsoft.ApplicationInsights.Web.ApplicationInsightsHttpModule, Microsoft.AI.Web" preCondition="managedHandler"/>
+		</modules>
+		<validation validateIntegratedModeConfiguration="false"/>
+	</system.webServer>
+	<system.codedom>
+		<compilers>
+		  <compiler language="c#;cs;csharp" extension=".cs" type="Microsoft.CodeDom.Providers.DotNetCompilerPlatform.CSharpCodeProvider, Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=2.0.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" warningLevel="4" compilerOptions="/langversion:default /nowarn:1659;1699;1701"/>
```

Comment:
I see you reformat - which is nice, 👍  for that, but the indention here is I think wrong - so I'd suggest either indent the whole file, or don't add indention changes at all.

Reply:
Not sure how those spaces (instead of a tab) slipped through the net. Fixed in 28aa9a2.